### ISSUE TITLE
replyTo should allow multiple addresses?

### ISSUE THREAD

**Issue by tomyam1** [nodemailer/nodemailer#1237]
1. What kind of issue are you reporting?
- [x] A bug in Nodemailer

2. Are you listed as a sponsor of Nodemailer project (see Sponsors button above)?
- [x] No. Unless it is a bug in Nodemailer you might find support from public forums like [StackOverflow](https://stackoverflow.com/search?q=nodemailer).

3. State your problem here:

This seems to be more a documentation issue because passing an array of addresses to `replyTo` does work as expected (tested with the SMTP and SES transports) but due to the docs stating otherwise the [typescript types](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/d4e72341390379eaa68b7c1567c941e693a43f3a/types/nodemailer/lib/mailer/index.d.ts#L108) only allow a single address.

The [API docs states](https://nodemailer.com/message/#routing-options) that `replyTo` is a single address.

```
replyTo - An email address that will appear on the Reply-To: field
```

This might be a mistake because:

[rfc5322](https://tools.ietf.org/html/rfc5322#section-3.6.2) does allow multiple addresses:
```
3.6.2.  Originator Fields
   ... In either case, an optional reply-to field MAY also be included, which contains
   the field name "Reply-To" and a comma-separated list of one or more
   addresses.
```
It seems also SES [allowes](https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/SES.html) `ReplyToAddresses` to be a list of addresses.

**Comment by andris9** [comment]
Supporting multiple Reply-To addresses is unintentional.

**Comment by sidharthv96** [comment]
@andris9 I'm also facing this issue where the behaviour is different from RFC.
RFC states that replyTo can be a list of emails just like to, cc, bcc.  
But nodemailer explicitly sets it as a single mail field [here](https://github.dev/nodemailer/nodemailer/blob/0b78689e5a8641dffb361dca4081c07b68d42498/lib/mailer/mail-message.js#L99-L105)

I can raise a PR to bring the behaviour in line with the RFC to allow multiple reply-to emails.

**Comment by andris9** [comment]
@sidharthv96 published v6.7.8 where there is no such constraint anymore, so you can use multiple Reply-To addresses
